client/app/components/composites/Menu/Menu.js

```diff
@@ -21,17 +21,24 @@ class Menu extends Component {
   constructor(props, context) {
     super(props, context);
 
+    this.handleMouseover = this.handleMouseover.bind(this);
+    this.handleMouseout = this.handleMouseout.bind(this);
     this.handleClick = this.handleClick.bind(this);
     this.handleBlur = this.handleBlur.bind(this);
     this.calculateDropdownPosition = this.calculateDropdownPosition.bind(this);
     this.state = {
       isOpen: false,
+      isHovering: false,
```

Comment:
Good point, I was just testing profile thing, since that's only testable thing on iOS world (our mobile layout is so wide).